stdlib/private/StdlibCollectionUnittest/CheckRangeReplaceableCollectionType.swift

```diff
@@ -862,6 +862,22 @@ self.test("\(testNamePrefix).removeSubrange()/semantics") {
   }
 }
 
+self.test("\(testNamePrefix).replaceSubrange()/closedRange/semantics") {
+  for test in removeRangeTests {
```

Comment:
What would you think about this?

```Swift
for test in removeRangeTests.filter { $0.rangeSelection != .emptyRange }
```

I personally find it a little bit clearer.